LOAN/LoansSpecific/LoanProducts.rdf

```diff
@@ -160,49 +172,49 @@
 		<skos:editorialNote>Future consideration: move this to ProductsAndServices ontology (fibo-fnd-pas-pas).</skos:editorialNote>
 	</owl:Class>
 	
-	<owl:Class rdf:about="&fibo-loan-typ-prod;LoanMarketCategory">
+	<owl:Class rdf:about="&fibo-loan-spc-prod;LoanMarketCategory">
 		<rdfs:subClassOf rdf:resource="&fibo-fnd-arr-cls;IndustrySectorClassifier"/>
 		<rdfs:label>loan market category</rdfs:label>
 		<skos:definition>category indicating the market domain in which the loan product is offered</skos:definition>
 	</owl:Class>
 	
-	<owl:NamedIndividual rdf:about="&fibo-loan-typ-prod;LoanMarketCategory-Automobile">
-		<rdf:type rdf:resource="&fibo-loan-typ-prod;LoanMarketCategory"/>
+	<owl:NamedIndividual rdf:about="&fibo-loan-spc-prod;LoanMarketCategory-Automobile">
+		<rdf:type rdf:resource="&fibo-loan-spc-prod;LoanMarketCategory"/>
 		<rdfs:label>auto loan</rdfs:label>
 	</owl:NamedIndividual>
 	
-	<owl:NamedIndividual rdf:about="&fibo-loan-typ-prod;LoanMarketCategory-Commercial">
-		<rdf:type rdf:resource="&fibo-loan-typ-prod;LoanMarketCategory"/>
+	<owl:NamedIndividual rdf:about="&fibo-loan-spc-prod;LoanMarketCategory-Commercial">
+		<rdf:type rdf:resource="&fibo-loan-spc-prod;LoanMarketCategory"/>
 		<rdfs:label>commercial loan</rdfs:label>
 	</owl:NamedIndividual>
 	
-	<owl:NamedIndividual rdf:about="&fibo-loan-typ-prod;LoanMarketCategory-Construction">
-		<rdf:type rdf:resource="&fibo-loan-typ-prod;LoanMarketCategory"/>
+	<owl:NamedIndividual rdf:about="&fibo-loan-spc-prod;LoanMarketCategory-Construction">
+		<rdf:type rdf:resource="&fibo-loan-spc-prod;LoanMarketCategory"/>
 		<rdfs:label>construction loan</rdfs:label>
 	</owl:NamedIndividual>
 	
-	<owl:NamedIndividual rdf:about="&fibo-loan-typ-prod;LoanMarketCategory-Education">
-		<rdf:type rdf:resource="&fibo-loan-typ-prod;LoanMarketCategory"/>
+	<owl:NamedIndividual rdf:about="&fibo-loan-spc-prod;LoanMarketCategory-Education">
```

Comment:
Student loan?

Reply:
@Pete - all of this needs to be revisited, in another pass.